webcrawler.py

```diff
@@ -5,12 +5,19 @@
 import scrapy
 from scrapy.crawler import CrawlerProcess
 from scrapy.http.request import Request
+import re
+
 
 KEYWORDS_FILE = 'textfile.txt'
 SCRAPY_OUTPUT_FILE = 'scrapy-output.json'
 RANKED_OUTPUT_FILE = 'output.csv'
 
 
+def contains_date(url):
+    """Returns True if the url contains a date."""
+    result= re.search(r'/(\d{4})/(\d{1,2})/(\d{1,2})/', url)
+    return bool(result)
```

Comment:
Nitpick: can turn into one line. There should also be a space on both sides of `=`.